examples/visualization/visualizers/segmentation.py

```diff
@@ -2,14 +2,14 @@
 import depthai as dai
 import numpy as np
 
-from depthai_nodes.ml.messages import SegmentationMasksSAM
+from depthai_nodes.ml.messages import SegmentationMask
 
 from .utils.colors import get_adas_colors, get_ewasr_colors, get_selfie_colors
-from .utils.message_parsers import parse_fast_sam_message, parse_segmentation_message
+from .utils.message_parsers import parse_segmentation_message
 
 
 def visualize_segmentation(
-    frame: dai.ImgFrame, message: dai.ImgFrame, extraParams: dict
+    frame: dai.ImgFrame, message: SegmentationMask, extraParams: dict
```

Comment:
I believe frame is actually np.ndarray type here and not ImgFrame. And same for all other `visualize_...` functions

Reply:
fixed in [31c4e62](https://github.com/luxonis/depthai-nodes/pull/127/commits/31c4e62bbfc7fb8aca0a41152619a3f16f3dbb94).